conformance/base/manifests.yaml

```diff
@@ -507,3 +507,144 @@ spec:
         resources:
           requests:
             cpu: 10m
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: grpc-infra-backend-v1
+  namespace: gateway-conformance-infra
+spec:
+  selector:
+    app: grpc-infra-backend-v1
+  ports:
+  - protocol: TCP
+    port: 8080
+    targetPort: 3000
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: grpc-infra-backend-v1
+  namespace: gateway-conformance-infra
+  labels:
+    app: grpc-infra-backend-v1
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: grpc-infra-backend-v1
+  template:
+    metadata:
+      labels:
+        app: grpc-infra-backend-v1
+    spec:
+      containers:
+      - name: grpc-infra-backend-v1
+	# TODO: Migate to gcr.io/k8s-staging-gateway-api once possible
```

Comment:
```suggestion
	# TODO: Migrate to gcr.io/k8s-staging-gateway-api once possible
```

Reply:
Also apparently we should have spaces instead of tabs here.

Reply:
I think this TODO should be gone by the time this is merged. How can I know before I've submitted this PR what tag CI will result in with the image build? Or will I have to split this into two PRs: docker image additions and test implementations?